Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_command_buffer.mm

```diff
@@ -1548,15 +1553,42 @@ inline void CheckPrimitiveType(MTLPrimitiveType primitiveType)
 #endif  // ANGLE_MTL_SIMULATE_DISCARD_FRAMEBUFFER
 }
 
+static bool shouldUseRasterizationRateMap(auto& cachedRenderPassDescObjC)
```

Comment:
@kkinnunen-apple I don't know how to better handle this so the rasterization map only gets set on the main render target, as I have no idea how to identify the main render target in an WebGL / WebXR application

We can't apply the rasterization rate map to offscreen passes and even this logic is broken if the user performs an offscreen pass which matches the double-width shared layout frame buffer size.